syntax = 'proto3';

service Trans{
    rpc output_trans(outputData) returns (inputData) {}
}


message outputData{}


message inputData{
    int32 idata = 1;
}